inc/enclosure.class.php

```diff
@@ -312,7 +312,7 @@ function rawSearchOptions() {
       ];
 
       $tab[] = [
-         'id'                 => '50',
```

Comment:
Since this SO already exists in a stable GLPI release, I guess we'll have to add a migration script (for display preferences, and maybe saved searches)